### DISCUSSION TITLE
求助Oracle 11g US7ASCII 中文乱码

### DISCUSSION THREAD

**Discussion by seanhuix** [dotnetcore/FreeSql#1466] [Q&A]
仔细看了文档中，安装 FreeSql.Provider.OracleOledb 使用 Oledb 驱动解决读取使用 US7ASCII 的 Oracle 数据库中文显示乱码问题。但还是乱码，还有别的注意事项吗？

**Comment by 2881099** [comment]
先百度一下，详细了解再回到 FreeSql Oledb
